src/components/ImageView/index.js

```diff
@@ -161,21 +161,31 @@ class ImageView extends PureComponent {
                             styles.w100,
                             styles.h100,
                         ]}
-                        resizeMode="center"
+                        resizeMode="contain"
                     />
                 </View>
             );
         }
-
         return (
             <View
                 ref={el => this.scrollableRef = el}
                 onLayout={(e) => {
                     const {width, height} = e.nativeEvent.layout;
-                    const imageWidth = this.state.imgWidth;
-                    const imageHeight = this.state.imgHeight;
+                    // eslint-disable-next-line react/no-access-state-in-setstate
+                    let imageWidth = this.state.imgWidth;
+                    // eslint-disable-next-line react/no-access-state-in-setstate
+                    let imageHeight = this.state.imgHeight;
```

Comment:
There's an edge case here - the first time an image is opened, it's not scaled properly.
steps to repro
1. Open chrome dev tools -> network tab -> disable cache. 
2. Open a small image. (eg: 1x1 pixel)

This is happening becuase `state.imgWidth` and `state.imgHeight` are `0` for the first time.
Let's fix this!

https://user-images.githubusercontent.com/29673073/153809449-94750ce8-26cb-4186-9a70-7e58ea2d6959.mp4

Reply:
fix that now @rushatgabhane 
we do the calculation before setImageRegion is called so I move the scall to setImageRegion and that works fine now.

Reply:
Nice find ^^